freqtrade/misc.py

```diff
@@ -113,3 +113,15 @@ def deep_merge_dicts(source, destination):
             destination[key] = value
 
     return destination
+
+
+def green(s):
+    return '\033[92m' + s + '\033[0m'
```

Comment:
I would want type-hints (at least) here ...
otherwise this is going to fail badly the moment someone has the idea to print a integer in green ...

In general, i'm not sure about doing this ourselfs.
 
Maybe we should simply depend on some tested library like [colorama](https://pypi.org/project/colorama/) or [termcolor](https://pypi.org/project/termcolor/) - since i guess the color-codes won't work on all terminal types the same way... and i doubt that we have much chances to test this ...

Reply:
That's borrowed from the scripts (get_market_pairs.py).

I was writing a VT100 emulator with DEC extensions from scratches 20 years ago so I still remember the sequences and the diffs between various terminals... Believe me -- it's just a subset of standard VT100 sequences, should (still) be supported on any physical or virtual terminal...

However, I'll take a look at colorama and termcolor, thanks...

Reply:
Ah, Windows... So colorama is needed, right.

Reply:
Confirmed that current version is working in windows